core/commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/util/Arrays.java

```diff
@@ -0,0 +1,132 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.util;
+
+import com.google.common.annotations.Beta;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.System.arraycopy;
+import static java.util.Arrays.copyOf;
+
+/**
+ * Utility methods to operate with arrays.
+ *
+ * @author Vlad Zhukovskiy
+ * @since 4.4.0
+ */
+@Beta
+public class Arrays {
```

Comment:
I would say that as this class is a "common class", this utility class is requiring tests to be sure that each method is working as expected.
For such class, 100% code coverage is expected